apis/v1alpha1/gateway_types.go

```diff
@@ -466,56 +448,47 @@ type RouteNamespaces struct {
 
 // GatewayAddress describes an address that can be bound to a Gateway.
 type GatewayAddress struct {
-	// Type of the Address. This is either "IPAddress" or "NamedAddress".
+	// Type of the address.
 	//
 	// Support: Extended
 	//
 	// +kubebuilder:default=IPAddress
 	Type AddressType `json:"type,omitempty"`
 
-	// Value. Examples: "1.2.3.4", "128::1", "my-ip-address". Validity of the
-	// values will depend on `Type` and support by the controller.
+	// Value of the address. The validity of the values will depend
+	// on the type and support by the controller.
```

Comment:
Nit: I think the backticks and capitalization above made it more clear that "type" was referring to an API field and not a characteristic of the controller.

Reply:
That would be find if it was used consistently. We can make a separate pass to add that.